client/src/components/ProfilePhoto/useStyles.ts

```diff
@@ -0,0 +1,60 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    marginTop: '10rem',
+    marginBottom: '10rem',
+    alignItems: 'center',
+    justifyContent: 'center',
+    minWidth: '20rem',
+  },
+  profileContainer: {
+    alignItems: 'center',
+    minHeight: '60vh',
+    margin: 'auto',
+  },
+  contentWrapper: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  title: {
+    marginTop: '5rem',
+  },
+  titleText: {
+    fontFamily: 'Roboto',
+    fontWeight: 'bolder',
+    fontSize: '1.5rem',
+    margin: 'auto',
+  },
+  avatar: {
+    marginTop: '4rem',
+    width: 150,
+    height: 150,
+  },
+  caption: {
+    marginTop: '2rem',
+    color: '#9e9e9e',
+    width: '10rem',
+    textAlign: 'center',
+  },
+  captionText: {
+    fontSize: '12px',
+  },
+  uploadBtn: {
+    marginTop: '2rem',
+    color: '#d32f2f',
+    fontSize: '12px',
+    lineHeight: '3rem',
+    minWidth: '20rem',
+  },
+  deleteBtn: {
+    color: '#9e9e9e',
```

Comment:
Ditto about the font size here.

Additionally to the typography, the palette of the app is another aspect of the theme and should ideally be consistent throughout the app. It would be better to use the `color` prop for `Button` components for these (Material UI theme is [customizable enough for most palette modifications](https://v4.mui.com/api/typography/#props)).

For when this isn't enough, it's a better alternative to create a reusable `MyCustomButton` component ([more info on this here](https://v4.mui.com/customization/components/#3-specific-variation-of-a-component).